CONTRIBUTING.md

```diff
@@ -60,6 +60,28 @@ Warning: Cannot update during an existing state transition (such as within `rend
 
 You can safely ignore these -- those won't show up to users on the production site.
 
+## Bypassing localhost HTTPS security issues
```

Comment:
One of the packages in https://github.com/OfficeDev/office-addin-scripts has support for setting the localhost bypass for Edge. This is used with add-in projects created with Yo Office to accomplish the same thing. It's automated and wouldn't require a user to do something.

Reply:
'office-addin-dev-settings appcontainer --loopback --yes" can be used on Windows to allow Edge locahost loopback.

Reply:
Thanks for your reference here, I will have a try with this setting solution.